cmd/validate/image.go

```diff
@@ -264,6 +265,11 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		a RFC3339 formatted value, e.g. 2022-11-18T00:00:00Z.
 	`))
 
+	cmd.Flags().BoolVar(&data.info, "info", data.info, hd.Doc(`
+	    Include additional information on the failures. For instance for policy
```

Comment:
Indent looks inconsistent here for some reason.

Reply:
Spaces instead of tabs. Thanks for spotting this!